forc-plugins/forc-doc/src/assets/FiraSans-LICENSE.txt

```diff
@@ -0,0 +1,94 @@
+Digitized data copyright (c) 2012-2015, The Mozilla Foundation and Telefonica S.A.
```

Comment:
Let's move each of these assets licenses into `forc-doc/licenses` directory so that they don't get compiled into the binary.

Let's also add a `forc-doc/licenses/README.md` that briefly mentions that the directory contains all of the licenses associated with the assets contained within the `assets` directory.